omnibot/services/slack/message.py

```diff
@@ -67,7 +67,7 @@ def _check_unsupported(self):
             logger.debug("ignoring message from slackbot", extra=self.event_trace)
             unsupported = True
         # Ignore threads
-        elif self.thread_ts:
+        elif self.thread_ts and not self._payload.get("bot", {}).get("bot_id") == "A06SS17UFFW":
```

Comment:
Is this a bot at Lyft? Maybe it would be better to make this a configurable option, then to include the list of bots to be ignored in the config instead?